docs/sources/tempo/configuration/_index.md

```diff
@@ -145,14 +145,20 @@ distributor:
         otlp:
             protocols:
                 grpc:
+                  # connections to Tempo running in a Docker container
+                  # endpoint: "0.0.0.0:4317"
                 http:
+                  # connections to Tempo running in a Docker container
```

Comment:
@joe-elliott Should this line also be removed, per your comment under gRPC?

Corrected this to sentence case, unless we need to remove the line. 
```suggestion
                  # Connections to Tempo running in a Docker container
```